test/plugins/test_autoimport.py

```diff
@@ -0,0 +1,236 @@
+from typing import Dict, List
```

Comment:
```suggestion
# Copyright 2022- Python Language Server Contributors.

from typing import Dict, List
```

Reply:
Fixed in 76f68de3a9b56c9f3ee014bee0da0bd141f8547a